.circleci/config.yml

```diff
@@ -125,6 +139,132 @@ jobs:
       - run-bazel-rbe:
           command: bazel test //test-end-to-end:test-end-to-end --test_output=streamed --spawn_strategy=standalone
 
+  test-assembly-mac-zip:
+    macos:
+      xcode: "9.0"
+    working_directory: ~/grakn
+    steps:
+      - checkout
+      - install-bazel-mac
+      - run:
+          command: |
+            bazel build //:assemble-mac-zip
+            unzip bazel-genfiles/grakn-core-all-mac.zip -d bazel-genfiles/
+            nohup bazel-genfiles/grakn-core-all-mac/grakn server start
+      - run: bazel test //test-deployment:test-deployment --test_output=streamed --spawn_strategy=standalone --cache_test_results=no
+      - run: bazel-genfiles/grakn-core-all-mac/grakn server stop
+
+# TODO: add after fixing the python issue regarding missing bdist_wheel
+#  test-assembly-windows-zip:
+#    machine: true
+#    working_directory: ~/grakn
+#    steps:
+#      - checkout
+#      - run:
+#          command: .circleci/test/assembly/windows-zip/windows-zip.py # TODO: test starting Grakn from a directory which has space
+#          no_output_timeout: 20m
+
+  test-assembly-linux-targz:
+    machine: true
+    working_directory: ~/grakn
+    steps:
+      - checkout
+      - install-bazel
+      - run:
+          command: |
+            bazel build //:assemble-linux-targz
+            tar -xf bazel-genfiles/grakn-core-all-linux.tar.gz -C bazel-genfiles
+            nohup bazel-genfiles/grakn-core-all-linux/grakn server start
+      - run: bazel test //test-deployment:test-deployment --test_output=streamed --spawn_strategy=standalone --cache_test_results=no
+      - run: bazel-genfiles/grakn-core-all-linux/grakn server stop
+
+  test-assembly-apt:
```

Comment:
I'm starting to see some inconsistencies in the namings of our distribution Bazel targets and CI jobs. Do you think we should make them consistent? 
E.g. should we rename this `test-assembly-linux-apt` because we have `test-assembly-linux-targz`? Or should we rename `test-assembly-linux-targz` to be `test-assembly-targz-linux`? The same question applies to `assemble-*` and `deploy-*` targets. I'm actually not so sure of the answer... 🤔

Reply:
Personally I think `test-assembly-linux-apt` makes sense as the naming becomes OS-centric: "this is a test for a Grakn distribution for Linux of which the format is Apt"

Reply:
That's what I'm feeling too. would this apply to build target names too, @lolski?

Reply:
I think so.

Reply:
okay cool, can you make the changes in this PR, @lolski?

Reply:
fixed in 54e8de318bec4742eb2b48e82a33c73fb263b41d